Mathlib/Analysis/Calculus/ContDiff.lean

```diff
@@ -2517,202 +2368,238 @@ theorem ContinuousLinearMap.norm_iteratedFderivWithin_le_of_bilinear (B : E →L
   have isoF : Fu ≃ₗᵢ[𝕜] F := LinearIsometryEquiv.ulift 𝕜 F
   have isoG : Gu ≃ₗᵢ[𝕜] G := LinearIsometryEquiv.ulift 𝕜 G
   -- lift `f` and `g` to versions `fu` and `gu` on the lifted spaces.
-  let fu : Du → Eu := isoE.symm ∘ f ∘ isoD
-  let gu : Du → Fu := isoF.symm ∘ g ∘ isoD
+  set fu : Du → Eu := isoE.symm ∘ f ∘ isoD with hfu
+  set gu : Du → Fu := isoF.symm ∘ g ∘ isoD with hgu
   -- lift the bilinear map `B` to a bilinear map `Bu` on the lifted spaces.
-  let Bu₀ : Eu →L[𝕜] Fu →L[𝕜] G
-  exact ((B.comp (isoE : Eu →L[𝕜] E)).flip.comp (isoF : Fu →L[𝕜] F)).flip
+  set Bu₀ : Eu →L[𝕜] Fu →L[𝕜] G := ((B.comp (isoE : Eu →L[𝕜] E)).flip.comp (isoF : Fu →L[𝕜] F)).flip
+    with hBu₀
   let Bu : Eu →L[𝕜] Fu →L[𝕜] Gu;
   exact
     ContinuousLinearMap.compL 𝕜 Eu (Fu →L[𝕜] G) (Fu →L[𝕜] Gu)
       (ContinuousLinearMap.compL 𝕜 Fu G Gu (isoG.symm : G →L[𝕜] Gu)) Bu₀
+  have hBu : Bu = ContinuousLinearMap.compL 𝕜 Eu (Fu →L[𝕜] G) (Fu →L[𝕜] Gu)
+      (ContinuousLinearMap.compL 𝕜 Fu G Gu (isoG.symm : G →L[𝕜] Gu)) Bu₀ := rfl
   have Bu_eq : (fun y => Bu (fu y) (gu y)) = isoG.symm ∘ (fun y => B (f y) (g y)) ∘ isoD := by
     ext1 y
-    simp only [Bu, ContinuousLinearMap.compL_apply, Function.comp_apply,
-      ContinuousLinearMap.coe_comp', LinearIsometryEquiv.coe_coe'', ContinuousLinearMap.flip_apply,
-      LinearIsometryEquiv.apply_symm_apply]
```

Comment:
Include the deleted
```lean
    simp only [Bu, ContinuousLinearMap.compL_apply, Function.comp_apply,
      ContinuousLinearMap.coe_comp', LinearIsometryEquiv.coe_coe'', ContinuousLinearMap.flip_apply,
      LinearIsometryEquiv.apply_symm_apply]
```
in a porting note?

Reply:
Done.